### PR TITLE
Add a `copyFragment` helper for plugin onCut/onCopy functions

### PR DESCRIPTION
**Pull request by justinweiss**
I took a stab at fixing #1410 and #1411 by creating an extra exposed helper function. Most of the new function was a direct move from `onCutOrCopy`, I just changed `fragment` to be a parameter.

### CODE REVIEW

**Comment by ianstormtaylor** [comment]
Hey @justinweiss this looks great to me!

Would you be down to rename the helper to `cloneFragment`? Since it's not actually performing the copy logic, just to be slightly more correct. Thanks!

**Comment by justinweiss** [comment]
Sure thing!

File: packages/slate-react/src/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 
 import Editor from './components/editor'
+import cloneFragment from './utils/copy-fragment'
```
Comment (ianstormtaylor) [inline packages/slate-react/src/index.js]: This one got missed I think and is causing the tests to tail. Once those are passing I'm down to merge! Thanks @justinweiss!

**Comment by ianstormtaylor** [comment]
Thanks @justinweiss!

**Pull request merged**